include/openthread.h

```diff
@@ -1271,6 +1271,28 @@ uint32_t otGetKeySequenceCounter(otInstance *aInstance);
 void otSetKeySequenceCounter(otInstance *aInstance, uint32_t aKeySequenceCounter);
 
 /**
+ * Get the thrKeySwitchGuardTime
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @returns The thrKeySwitchGuardTime value.
+ *
+ * @sa otSetKeySwitchGuardTime
+ */
+uint32_t otGetKeySwitchGuardTime(otInstance *aInstance);
+
+/**
+ * Set the thrKeySequenceCounter.
+ *
+ * @param[in]  aInstance            A pointer to an OpenThread instance.
+ * @param[in]  aGuardTime           The thrKeySwitchGuardTime value.
```

Comment:
I think it is worth updating the comments to indicate the units here (same for above). Also the comment parameter name doesn't match the actual parameter.